pkg/scheduler/api/job_info.go

```diff
@@ -415,3 +415,9 @@ func (ji *JobInfo) Pipelined() bool {
 
 	return occupied >= ji.MinAvailable
 }
+
+func (ji *JobInfo) Preemptable() bool {
```

Comment:
move this to related plugin, e.g. `ji.MinAvailable <= (occupied - 1)` does not mean preemptable for drf plugin.